yarn-project/circuits.js/src/structs/complete_address.ts

```diff
@@ -22,71 +22,93 @@ export class CompleteAddress {
   public constructor(
```

Comment:
With key rotations, won't the wallets actually get wrecked as many of them currently are just calling the account which seems to just have one value there 🤔 ☠️ Or is the complete address only really interesting for the "deployment" of it but not if rotated 🤔

Reply:
Yes, key rotation will fuck it up. i think we should address that simply by having an endpoint on pxe and requesting the info from there when needed --> and pxe would grab it from key registry or from local db